electron/utils.ts

```diff
@@ -142,3 +152,28 @@ export function askActionConfirmation({
 
   return choice === 0;
 }
+
+export const initNucleus = (isDev: boolean, app: any) => {
+  Nucleus.init('6218cf3ef5e5d2023724d89b', {
```

Comment:
@erdkse what id is this?

Reply:
This is my accounts credential - We will delete this when we move to Kubeshop's account